example/main.tf

```diff
@@ -0,0 +1,32 @@
+terraform {
+  required_providers {
+    wavefront = {
+      source  = "vmware/wavefront",
+      version = ">= 3.0.0"
```

Comment:
Is example a new directory you are creating? Usually example terraform files go in the docs folder under docs/resources.

Reply:
Yes, this is a new directory I created. Just want to provide some examples, so that we can use them to test locally. Will move it to the docs/resources then

Reply:
Oh, I think the example folder I created served as a different use case. I created so that a dev who fork this repo, can directly run terraform in side this folder, instead of copy example resource in the docs/resources.